java/vector/src/test/java/org/apache/arrow/vector/ipc/TestArrowFile.java

```diff
@@ -131,4 +150,67 @@ public void testFileStreamHasEos() throws IOException {
       }
     }
   }
+
+  @ParameterizedTest
+  @MethodSource("dictionaryParams")
+  public void testMultiBatchDictionaries(int state) throws Exception {
+    File file = new File("target/mytest_multi_batch_dictionaries_" + state + ".arrow");
```

Comment:
This wouldn't work right if the test got built as a JAR. These inputs should probably be moved to test resoucres and opened using getResource(). This could be a separate issue.

Reply:
good point, I was just copying previous tests. I'll file an issue.